www/c-icap/src/opnsense/service/conf/actions.d/actions_cicap.conf

```diff
@@ -0,0 +1,35 @@
+[start]
+command:/usr/local/opnsense/scripts/OPNsense/CICAP/setup.sh;/usr/local/etc/rc.d/c-icap start
+parameters:
+type:script
+message:starting c-icap
+
+[stop]
+command:/usr/local/etc/rc.d/c-icap stop; exit 0
+parameters:
+type:script
+message:stopping c-icap
+
+[restart]
+command:/usr/local/opnsense/scripts/OPNsense/CICAP/setup.sh;/usr/local/etc/rc.d/c-icap restart
+parameters:
+type:script
+message:restarting c-icap
+
+[reconfigure]
+command:/usr/local/opnsense/scripts/OPNsense/CICAP/setup.sh;/usr/local/etc/rc.d/c-icap restart
+parameters:
+type:script
+message:reconfigure c-icap
+
+[status]
+command:/usr/local/etc/rc.d/c-icap status;exit 0
+parameters:
+type:script_output
+message:request c-icap status
+
+[checkclamav]
+command:/usr/local/sbin/configctl firmware plugin clamav
+parameters:
+type:script
+message:check if ClamAV is installed
```

Comment:
@fabianfrz You mean with parameters % and call via parameters via Controller? 
@fichtner I asked Ad for an easy way, but I have no idea to build a hide function via API :)